cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -303,24 +368,40 @@ func IsKubernetesWorkerQueueInputValid(requestedWorkerQueue *astro.WorkerQueue)
 // QueueExists takes a []existingQueues and a queueToCreate as arguments
 // It returns true if queueToCreate exists in []existingQueues
 // It returns false if queueToCreate does not exist in []existingQueues
-func QueueExists(existingQueues []astro.WorkerQueue, queueToCreate *astro.WorkerQueue) bool {
+func QueueExists(existingQueues []astroplatformcore.WorkerQueue, queueToCreate *astroplatformcore.WorkerQueueRequest, queueToCreateOrUpdateHybrid *astroplatformcore.HybridWorkerQueueRequest) bool {
```

Comment:
the function param and comments here are not aligned, we should either have them all be 
`queueToCreateOrUpdateHybrid` and `queueToCreateOrUpdate` or
`queueToCreateHybrid` and `queueToCreate`

Reply:
fixed